cmd/horcrux/cmd/key2shares_test.go

```diff
@@ -25,41 +28,91 @@ func TestKey2Shares(t *testing.T) {
 	}{
 		{
 			name:      "valid threshold and shares",
-			args:      []string{privValidatorKeyFile, "2", "3"},
+			args:      []string{testChainID, privValidatorKeyFile, "2", "3"},
 			expectErr: false,
 		},
 		{
 			name:      "valid threshold and shares 2",
-			args:      []string{privValidatorKeyFile, "3", "5"},
+			args:      []string{testChainID, privValidatorKeyFile, "3", "5"},
 			expectErr: false,
 		},
 		{
 			name:      "threshold exactly half of shares",
-			args:      []string{privValidatorKeyFile, "2", "4"},
+			args:      []string{testChainID, privValidatorKeyFile, "2", "4"},
 			expectErr: true,
 		},
 		{
 			name:      "threshold less than half of shares",
-			args:      []string{privValidatorKeyFile, "1", "3"},
+			args:      []string{testChainID, privValidatorKeyFile, "1", "3"},
 			expectErr: true,
 		},
 		{
 			name:      "threshold exceeds shares",
-			args:      []string{privValidatorKeyFile, "4", "3"},
+			args:      []string{testChainID, privValidatorKeyFile, "4", "3"},
 			expectErr: true,
 		},
 		{
 			name:      "non-numeric threshold and shares",
-			args:      []string{privValidatorKeyFile, "two", "three"},
+			args:      []string{testChainID, privValidatorKeyFile, "two", "three"},
+			expectErr: true,
+		},
+	}
+
+	cwd, err := os.Getwd()
+	require.NoError(t, err)
+
+	require.NoError(t, os.Chdir(tmp))
```

Comment:
It doesn't look like changing the directory is necessary, unless there is something dependent about the working directory for the command. 

It's unusual. If it is required, the change back to the original directory should be in a `t.Cleanup` call so that there isn't a risk of the call being missed during an intermediate failure.

Reply:
These create shares commands output the files to the cwd. Would be nice to have a flag to change the output dir so that the tests can avoid this. Will modify

Reply:
Flag added, test updated